src/robotjs.cc

```diff
@@ -26,7 +26,66 @@ int keyboardDelay = 10;
 
 */
 
-NAN_METHOD(moveMouse) 
+int CheckMouseButton(const char * const b, MMMouseButton * const button)
```

Comment:
Added this new function, following the style of your CheckKeyCodes, and changed the other mouse functions to also use it.